traitsui/tests/editors/test_text_editor.py

```diff
@@ -133,6 +133,26 @@ def test_text_editor_custom_style_placeholder(self):
             else:
                 self.assertEqual(placeholder, "Enter name")
 
+    def test_clear_button(self):
+        foo = Foo()
+        view = View(
+            Item(
+                name="name",
+                style="simple",
+                editor=TextEditor(clear_button=True),
+            )
+        )
+        tester = UITester()
+        with tester.create_ui(foo, dict(view=view)) as ui:
+            name_editor, = ui.get_editors("name")
+            try:
+                clear_button = name_editor.control.isClearButtonEnabled()
+            except AttributeError:
```

Comment:
Might be better to pre-test with `hasattr` or test that `AttributeError` is because of `isClearButtonEnabled` and raise otherwise.